app/views/users/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= pluralize(@user.errors.count, "error") %>
+      prohibited this user from being saved:
+    </h2>
+
+    <ul>
+    <% @user.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+    </ul>
+  </div>
+<% end %>
+<%= form_for @user, url: user_path(@user) do |usr| %>
```

Comment:
Missing the commented out `form_tag` that was required prior to this. Commented out code is normally requested to be removed from the repo according to the guidelines but this project in particular requires this one to have that as commented out code.

[REQUIRED]